mappings/net/minecraft/packet/OpenScreenPacket.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS iw net/minecraft/packet/OpenScreenPacket
```

Comment:
`S2C` or `C2S`? Or is this used by both client or server?

Reply:
must be server sent client bound

Reply:
Seems to be handled by the client and then the client sets all its stuff to the packets screen, so S2C.